Functions.Templates/Templates/BlobTrigger-CSharp-Isolated/.template.config/template.json

```diff
@@ -51,7 +51,8 @@
             "ManualInstructions": [],
             "args": {
                 "referenceType": "package",
-            "reference": "Microsoft.Azure.Functions.Worker.Extensions.Storage", "version": "4.0.4",
+                "reference": "Microsoft.Azure.Functions.Worker.Extensions.Storage",
+                "version": "5.1.1-preview2",
                 "projectFileExtensions": ".csproj"
             }
         },
```

Comment:
@fabiocav or @soninaren - is it safe to bump the identity here?

`"identity": "Azure.Function.CSharp.Isolated.BlobTrigger.3.x",` -> `"identity": "Azure.Function.CSharp.Isolated.BlobTrigger.4.x",`